webroot/js/toolbar.js

```diff
@@ -29,7 +37,13 @@
 		}
 		var body = doc.body;
 		iframe = doc.createElement('iframe');
-		iframe.setAttribute('style', 'position: fixed; bottom: 0; right: 0; border: 0; outline: 0; overflow: hidden; z-index: 99999;');
+        iframe.style.position = 'fixed';
+        iframe.style.bottom = 0;
+        iframe.style.right = 0;
+        iframe.style.border = 0;
+        iframe.style.outline = 0;
+        iframe.style.overflow = 'hidden';
+        iframe.style.zIndex = 99999;
```

Comment:
Yeah exactly @markstory . I tried it with setAttribute, but it won't work. You can just change styles with directly accessing the properties separated with e.g. ```iframe.style.position``` .